### PR TITLE
Contact tracing (ES)

### PR DESCRIPTION
**Pull request by nicoonswift**
Hello,

Here a Spanish translation proposal for the [Contact Tracing](https://nshipster.com/contact-tracing/) article.

### CODE REVIEW

File: 2020-04-13-contact-tracing.md [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+status:
+  swift: 5.2
+revisions:
+  "2020-04-13": First Publication
+  "2020-04-29": Updated for Exposure Notification v1.2
+---
+
+> Más vale prevenir que lamentar.
+
+Una intervención temprana es una de las estrategias más efectivas para el trato de enfermedades. Lo cual no solo aplica al cuerpo humano, si no que también para la sociedad en conjunto. Es por eso que funcionarios de la salud pública usan el rastreo de contactos como su primera línea de defensa contra la propagación de enfermedades contagiosas en una población.
```
Comment (jfsagasti) [inline 2020-04-13-contact-tracing.md:27]: What about...

"no solo se aplica al cuerpo humano, sino también a la sociedad en su conjunto." ?

Comment (nicoonguitar) [reply 2020-04-13-contact-tracing.md]: Hello, agreed. Wording changed on latest commit.

File: 2020-04-13-contact-tracing.md [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+
+Estos días, mucho escuchamos acerca del rastreo de contactos, pero la técnica ha sido utilizada durante décadas. Lo que ha cambiado es que, gracias a la ubicuidad de los dispositivos electrónicos personales, podemos automatizar lo que antes era - hasta hoy - un proceso manual e intensivo. Así como un "computador" solía ser un puesto de trabajo para humanos, el rol the un "rastreador de contactos" podría pronto ser realizado principalmente por aplicaciones.
+
+El 10 de abril, Apple y Google [anunciaron][apple press release] en conjunto una iniciativa para desplegar, en los próximos meses, la funcionalidad de rastreo de contactos en billones de dispositivos Android e iOS. Como parte de este anuncio, las compañías compartieron bosquejos de las especificaciones para la [criptografía][cryptography specification], [hardware][hardware specification] y [software][software specification] involucrados en su solución propuesta.
```
Comment (jfsagasti) [inline 2020-04-13-contact-tracing.md:31]: Are those [...] links? If so, URLs are missing :O

File: 2020-04-13-contact-tracing.md [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+
+En este artículo, vamos a dar una primera mirada a estas especificaciones - particularmente al framework `ExposureNotification` propuesto por Apple - y usaremos lo aprendido para anticipar como se verá esto una vez puesto en práctica.
```
Comment (jfsagasti) [inline 2020-04-13-contact-tracing.md:33]: vamos a echar un primer vistazo?

File: 2020-04-13-contact-tracing.md [hunk 4]
```diff
@@ -0,0 +1,465 @@
...
+
+El 29 de abril, Apple lanzó iOS 13.5 beta 1, el cual incluye el primer release del framework `ExposureNotification` (previamente `ContactTracing`). El contenido de este artículo ha sido actualizado para reflejar estos cambios.
+
+{% endwarning %}
+
+* * *
+
+## ¿Qué es el rastreo de contactos?
+
+<dfn>El rastreo de contactos</dfn> es una técnica utilizada por los funcionarios de salud pública
```
Comment (jfsagasti) [inline 2020-04-13-contact-tracing.md:45]: These artificial breaks are difficult to read in the translation and make things too literal. I'd prefer a less literal translation, not tied to the need of doing a line break and translating each fragment.

File: 2020-04-13-contact-tracing.md [hunk 5]
```diff
@@ -0,0 +1,465 @@
...
+Con más de un millón de casos confirmados de <abbr>COVID-19</abbr> en todo el mundo, 
+muchas regiones ya han pasado el punto en el cual el rastreo de contactos es práctico.
+Pero eso no quiere decir que no pueda desempeñar un papel esencial
+en las próximas semanas y meses.
+
+## "Solo Apple <ins>(y Google)</ins> pueden hacerlo"
+
+Desde el brote, varios [gobiernos](https://www.pepp-pt.org) y [académicos](https://github.com/DP-3T/documents) han propuesto estándares para el rastreo de contactos. Pero el desarrollo más significativo hasta ahora fue ayer con el anuncio de Apple y Google de una iniciativa en conjunto.
+
+De acuerdo al <abbr title="United Kingdom National Health Service">NHS</abbr>, cerca del 60% de adultos de una población necesitarían participar para que el rastreo de contactos digital sea efectivo. Investigadores de las instituciones mencionadas anteriormente han notado que los límites impuesto por iOS para las aplicaciones de terceros hacen que este nivel de participación sea poco probable.
```
Comment (jfsagasti) [inline 2020-04-13-contact-tracing.md:79]: límites impuestos

File: 2020-04-13-contact-tracing.md [hunk 6]
```diff
@@ -0,0 +1,465 @@
...
+Pero eso no quiere decir que no pueda desempeñar un papel esencial
+en las próximas semanas y meses.
+
+## "Solo Apple <ins>(y Google)</ins> pueden hacerlo"
+
+Desde el brote, varios [gobiernos](https://www.pepp-pt.org) y [académicos](https://github.com/DP-3T/documents) han propuesto estándares para el rastreo de contactos. Pero el desarrollo más significativo hasta ahora fue ayer con el anuncio de Apple y Google de una iniciativa en conjunto.
+
+De acuerdo al <abbr title="United Kingdom National Health Service">NHS</abbr>, cerca del 60% de adultos de una población necesitarían participar para que el rastreo de contactos digital sea efectivo. Investigadores de las instituciones mencionadas anteriormente han notado que los límites impuesto por iOS para las aplicaciones de terceros hacen que este nivel de participación sea poco probable. 
+
+Por un lado, se siente extraño de felicitar a Apple por dar una paso adelante en la resolución de un problema que crearon ellos mismos. Pero podemos estar de acuerdo que este anuncio es algo para celebrar. No es una exageración decir que esto no sería posible sin su ayuda.
```
Comment (jfsagasti) [inline 2020-04-13-contact-tracing.md:81]: es extraño felicitar...

por dar **un** paso.

Pero sin duda este acuerdo es algo que celebrar. Y no es exagerado decir que esto no sería posible sin su ayuda.

File: 2020-04-13-contact-tracing.md [hunk 7]
```diff
@@ -0,0 +1,465 @@
...
+
+## ¿Qué proponen Apple and Google como una solución?
+
+A un nivel alto, Apple y Google están proponiendo un estándar común sobre como los dispositivos electrónicos personales (teléfonos, tablets y relojes) pueden automatizar el proceso de rastreo de contactos.
```
Comment (jfsagasti) [inline 2020-04-13-contact-tracing.md:85]: A grandes rasgos, Apple y Google ...

File: 2020-04-13-contact-tracing.md [hunk 8]
```diff
@@ -0,0 +1,465 @@
...
+
+En lugar de que los trabajadores de salud deban llamar a los contactos por teléfono,
```
Comment (jfsagasti) [inline 2020-04-13-contact-tracing.md:87]: Review line breaks

File: 2020-04-13-contact-tracing.md [hunk 9]
```diff
@@ -0,0 +1,465 @@
...
+comenzando con la
+[criptografía][cryptography specification] (derivación y rotación de claves),
+seguido por el
+[hardware][hardware specification] (Bluetooth)
+y el
+[software][software specification] (app).
+
+### Criptografía
+
+Cuando instalas una aplicación y la abres por primera vez,
```
Comment (jfsagasti) [inline 2020-04-13-contact-tracing.md:116]: Review line breaks

**Comment by jfsagasti** [comment]
Thank you Nico! I've left some comments. Please review all the forced line breaks (we are not doing those in the Spanish translations).

File: 2020-04-13-contact-tracing.md (hunk 3 above)
Comment (nicoonguitar) [reply 2020-04-13-contact-tracing.md:33]: What about "Vamos a dar un primer vistazo" ? Sounds a bit more formal

Comment (jfsagasti) [reply 2020-04-13-contact-tracing.md:33]: Nice!

**Comment by nicoonguitar** [comment]
> Thank you Nico! I've left some comments. Please review all the forced line breaks (we are not doing those in the Spanish translations).

@jfsagasti Hello, I pushed the changes. I also fixed a couple of broken links I had 👍

File: 2020-04-13-contact-tracing.md [hunk 10]
```diff
@@ -0,0 +1,298 @@
...
+
+### Hardware
+
+Bluetooth organiza las comunicaciones entre dispositivos bajo el concepto de <dfn>servicios</dfn>.
+
+Un servicio describe un conjunto de características para realizar una tarea en particular. Un dispositivo puede comunicarse con múltiples servicios en el transcurso de su operación. Muchas definiciones de servicio son [estandarizadas](https://www.bluetooth.com/specifications/gatt/) para que los dispositivos que hacen el mismo tipo de cosas se comuniquen de la misma manera.
+
+Por ejemplo, un monitor inalámbrico de frecuencia cardíaca que usa Bluetooth para comunicarse con su teléfono tendría un perfil que contiene dos servicios: un servicio primario de frecuencia cardíaca y un servicio secundario de batería.
+
+El estándar de rastreo de contactos de Apple y Ggoogle define un nuevo servicio de Detección de Contactos (en inglés: Contact Detection)
```
Comment (jfsagasti) [inline 2020-04-13-contact-tracing.md:130]: Google

Comment (nicoonguitar) [reply 2020-04-13-contact-tracing.md]: Fixed !

**Comment by jfsagasti** [comment]
Thank you Nico! Feel free to merge it when you consider it :)

**Review by jfsagasti — APPROVED** [review]

**Comment by nicoonguitar** [comment]
> Thank you Nico! Feel free to merge it when you consider it :)

Great news ! But I can't do so, I don't have write access:
> Only those with write access to this repository can merge pull requests.

**Comment by nicoonswift** [comment]
@jfsagasti Hello, besides the translation, are there any other steps I need to do on my contribution in order to see the article published in https://nshipster.es ?

**Comment by jfsagasti** [comment]
@nicoonguitar Hi! The merge should have triggered the publication if I am not wrong. Could you check @mattt? :)

**Comment by nicoonswift** [comment]
> @nicoonguitar Hi! The merge should have triggered the publication if I am not wrong. Could you check @mattt? :) 

Hello, as today I still don’t see the publication, a step could be missing there CC @mattt  . 

Happy to do future translation contributions.

**Comment by nicoonswift** [comment]
> @jfsagasti Hello, besides the translation, are there any other steps I need to do on my contribution in order to see the article published in https://nshipster.es ? 

Hello @mattt, a reminder on this contribution, along #20 it didn't make it to be published.

**Pull request merged**